vue-components/src/components/Button.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+	<button :class="[ 'wikit', 'wikit-Button', `wikit-Button--${ flavor }` ]">
+		<slot />
+	</button>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+/**
+ * A Button
+ */
+export default Vue.extend( {
+	name: 'Button',
+	props: {
+		/**
+		 * The flavor of the button
+		 */
+		flavor: {
+			type: String,
+			// TODO why does the shorter version cause problems with doc-gen?
+			validator( value: string ): boolean {
+				return [ 'neutral' ].includes( value );
+			},
+			default: 'neutral',
+		},
+	},
+} );
+</script>
+
+<style lang="scss">
+$wikit-Button-medium-padding-vertical: $dimension-padding-vertical-medium;
+$wikit-Button-medium-padding-horizontal: $dimension-padding-horizontal-medium;
+
+$wikit-Button-large-padding-vertical: $dimension-padding-vertical-large;
+$wikit-Button-large-padding-horizontal: $dimension-padding-horizontal-large;
+
+$wikit-Button-font-family: $font-family-style-label;
+$wikit-Button-font-weight: $font-weight-bold;
+$wikit-Button-font-size: $font-size-style-label; // unit? px vs em vs rem
+$wikit-Button-line-height: $font-line-height-style-label;
+$wikit-Button-border-width: $border-width-thin;
+$wikit-Button-border-style: $border-style-base;
+$wikit-Button-border-radius: $border-radius-base;
+
+/**
+ * Construction site identified:
+ * How do we name the component level token?
+ *
+ * Suggestion for naming scheme:
+ * wikit-<Component>-[prop]-[state]-css-property
+ */
+$wikit-Button-default-cursor: $cursor-pointer;
+$wikit-Button-disabled-cursor: $cursor-default;
+
+.wikit-Button {
+	padding-inline: $wikit-Button-medium-padding-horizontal;
+	padding-block: $wikit-Button-medium-padding-vertical;
+	font-family: $wikit-Button-font-family;
+	font-weight: $font-weight-bold;
+	font-size: $font-size-style-label;
```

Comment:
If we have the component level tokens above ($wikit-Button-font-weight and $wikit-Button-font-size) why not use them?

Reply:
Whoops, that was an accident. Fixed!